xclim/testing/tests/test_utils.py

```diff
@@ -107,10 +107,10 @@ def test_open_testdata():
 def test_list_datasets():
     out = list_datasets()
 
-    assert out.columns == ["size", "url"]
+    assert list(out.columns) == ["size", "url"]
     np.testing.assert_allclose(
-        out.loc[
-            "cmip6/o3_Amon_GFDL-ESM4_historical_r1i1p1f1_gr1_185001-194912.nc"
-        ].size,
-        0.825216,
+        out.loc["cmip6/o3_Amon_GFDL-ESM4_historical_r1i1p1f1_gr1_185001-194912.nc"][
+            "size"
+        ],
+        845.021484375,
```

Comment:
What changed here? Those figures are quite the departure.

Reply:
haha, I wrote the test in MB, but then I changed to code to return KB but forgot to change the value here!